charts/fluid-dataloader/juicefs/templates/configmap.yaml

```diff
@@ -0,0 +1,33 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ printf "%s-data-load-script" .Release.Name }}
+  labels:
+    release: {{ .Release.Name }}
+    role: dataload-job
+data:
+  dataloader.distributedLoad: |
+    #!/usr/bin/env bash
+    set -xe
+
+    function main() {
+        paths="$DATA_PATH"
+        paths=(${paths//:/ })
+
+        podNames="$POD_NAMES"
+        podNames=(${podNames//:/ })
+
+        ns="$POD_NAMESPACE"
+        for((i=0;i<${#podNames[@]};i++)) do
+          local pod="${podNames[i]}"
+
+          for((j=0;j<${#paths[@]};j++)) do
+            echo -e "juicefs warmup on $pod ${paths[j]} starts"
+            /usr/local/bin/kubectl -n $ns exec -it $pod -- $COMMAND
```

Comment:
Will  you add `timeout` to avoid the job hang?

Reply:
done